src/main/java/org/opensearch/security/privileges/ProtectedIndexAccessEvaluator.java

```diff
@@ -73,31 +73,29 @@ public PrivilegesEvaluatorResponse evaluate(
         final String action,
         final IndexResolverReplacer.Resolved requestedResolved,
         final PrivilegesEvaluatorResponse presponse,
-        final SecurityRoles securityRoles
+        final Set<String> mappedRoles
```

Comment:
This is also a good change on its own if you wanted to pull it out into another PR

Reply:
+1